### ISSUE TITLE
Possible to call remotely?

### ISSUE THREAD

**Issue by twf-nikhila** [as-a-service/screenshot#8]
Is there a way to call this remotely from another application and it returns the image content in some formats like base64?

**Comment by steren** [comment]
Yes, that's that the container is for. Deploy it to Cloud Run and send a request to it to receive the image bytes
